src/interaction.ts

```diff
@@ -0,0 +1,83 @@
+// export interface changeSet {
+//     ResponseMetadata: DescribeChangeSetOutput;
+//     ChangeSetName: string;
+//     ChangeSetId: string;
+//     StackId: string;
+//     StackName: string;
+//     Parameters: [];
+//     CreationTime: Date;
+//     ExecutionStatus: string;
+//     Status: string;
+//     NotificationARNs: [];
+//     RollbackConfiguration: {};
+//     Capabilities: [];
+//     Tags: [];
+//     Changes: [];
+//     StatusReason?: any;
+//     // DescribeChangeSetOutput: any;
+// }
+
+import Table from "cli-table";
+import * as readline from "readline";
+
+// let rl = readline.createInterface({
+//     input: process.stdin,
+//     output: process.stdout,
+// });
+
+export class Interaction {
+    private rl: readline.Interface;
+
+    // readlie
+    constructor() {
+        this.rl = readline.createInterface({
+            input: process.stdin,
+            output: process.stdout,
+        });
+    }
+
+    close() {
+        this.rl.close();
+    }
+
+    // pass in readline
+    async confirmChanges(changeSet: any): Promise<boolean> {
+        const table = new Table({
+            head: ["Type", "Action", "Logistical Resource Id", "Resource Type"],
+            colWidths: [10, 20, 30, 30],
```

Comment:
This should be set dynamically

Reply:
I use Prettier as VScode plugin, do you mean that the table's columns should be dynamic?

Reply:
ya, the table should fit the console window